app/src/AppRoutes.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Suspense } from 'react';
```

Comment:
Renamed `Routes` component to `AppRoutes` to avoid conflict with `react-router.Routes`.